config/install/configure/observability/kustomization.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - github.com/Kuadrant/gateway-api-state-metrics/config/kuadrant?ref=0.5.0
```

```suggestion
  - github.com/Kuadrant/gateway-api-state-metrics/config/kuadrant?ref=0.6.0
```

@maleck13 I've pushed a new release of this with the v1 api versions.

---

Related change in a couple other places https://github.com/Kuadrant/kuadrant-operator/pull/1016